code/game/objects/items/weapons/storage/stickersheets.dm

```diff
@@ -16,6 +16,22 @@
 		/obj/item/sticker
 	)
 
+// List/Procs to list all stickers for custom stickersheet(similar to lunchbox code)
+var/list/stickersheet_stickers_ = typesof(/obj/item/sticker)
```

Comment:
This needs to be a GLOB list, and have a DMDoc
Also, the name ends with an underscore, which it shouldn't